cms_form_compassion/models/invoice.py

```diff
@@ -0,0 +1,15 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    @author: Robin Berguerand <robin.berguerand@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+    auto_cancel_no_transaction = fields.Boolean(default=False, help='If true, cancel the invoice if the linked payment '
+                                                        'transaction is cancelled',oldname='cancel_invoice')
```

Comment:
It doesn't look like "cancel_invoice" was an existing field. We should either remove this old field if we find it or remove this attribute.
```suggestion
                                                        'transaction is cancelled')
```